### ISSUE TITLE
Posix time zone format in TZ.h

### ISSUE THREAD

**Issue by gmag11** [esp8266/Arduino#8423]
Hi,

Some of the entries on [TZ.h](https://github.com/esp8266/Arduino/blob/14c3798d4ff959254599f2904579412acba3f175/cores/esp8266/TZ.h) like https://github.com/esp8266/Arduino/blob/14c3798d4ff959254599f2904579412acba3f175/cores/esp8266/TZ.h#L72 use a notation which I haven't found in several references like https://developer.ibm.com/articles/au-aix-posix/.

Could this be a mistake in [source](https://raw.githubusercontent.com/nayarsystems/posix_tz_db/master/zones.csv)? Where is the official specification of this format?

**Comment by mhightower83** [comment]
This is the quoted format. 

See https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap08.html

Look at the `TZ` section or search for "two formats". 
I must admit I do find their description hard to follow. 

You will also find this quoted format use on Linux systems for some of the `TZ` strings.
On Ubuntu you can find a list of binary TZ strings in the directory `/usr/share/zoneinfo/posix`. 
While the files in this tree are binary; you can display the interesting text portion with:
```
tail -1 America/Argentina/Buenos_Aires
```
The last line of the binary reveals the `TZ `string: `<-03>3`

This will also works for the system configures value.
```
tail -1 /etc/localtime
```
